test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperFormExtensionsTest.cs

```diff
@@ -397,10 +397,14 @@ public void BeginFormWithMethodParameter_CallsHtmlGeneratorWithExpectedValues(Fo
                     null))  // htmlAttributes
                 .Returns(tagBuilder)
                 .Verifiable();
-            htmlGenerator
-                .Setup(g => g.GenerateAntiforgery(htmlHelper.ViewContext))
-                .Returns(HtmlString.Empty)
-                .Verifiable();
+
+            if (method != FormMethod.Get)
+            {
+                htmlGenerator
+                    .Setup(g => g.GenerateAntiforgery(htmlHelper.ViewContext))
+                    .Returns(HtmlString.Empty)
+                    .Verifiable();
+            }
```

Comment:
Please split. And if we're doing the same test multiple times, feed free to nuke some bits (but include PR comments so we don't have to ask why something died an ignoble death).